src/Winium.Desktop.Driver/CommandExecutors/GetElementAttributeExecutor.cs

```diff
@@ -58,6 +60,18 @@ private static object PrepareValueToSerialize(object obj)
                 return controlType.ProgrammaticName;
             }
 
+            var intArray = obj as Int32[];
+            if (intArray != null)
+            {
+                return string.Join(",", intArray);
```

Comment:
Why do integer arrays need special serialization? By default the should be serialized as json object like other arrays, dictionaries, etc.